apps/web/src/views/BuyCrypto/components/ProviderCampaign/ProviderCampaign.tsx

```diff
@@ -0,0 +1,45 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Box, Flex, Text } from '@pancakeswap/uikit'
+import getTimePeriods from '@pancakeswap/utils/getTimePeriods'
+import Image from 'next/image'
+import { CURRENT_CAMPAIGN_TIMESTAMP, ONRAMP_PROVIDERS } from 'views/BuyCrypto/constants'
+import pocketWatch from '../../../../../public/images/pocket-watch.svg'
+
+const activeCampaigns: { [provider in keyof typeof ONRAMP_PROVIDERS]: boolean } = {
+  [ONRAMP_PROVIDERS.Mercuryo]: false,
+  [ONRAMP_PROVIDERS.MoonPay]: false,
+  [ONRAMP_PROVIDERS.Transak]: false,
+}
+
+const ProviderCampaign = ({ provider }: { provider: keyof typeof ONRAMP_PROVIDERS }) => {
+  const { t } = useTranslation()
+
+  const currentTimestamp = Math.floor(Date.now() / 1000)
+  const {
+    days,
+    hours,
+    minutes,
+    seconds: campaignEndTimeInSeconds,
+  } = getTimePeriods(currentTimestamp - CURRENT_CAMPAIGN_TIMESTAMP)
+
+  return (
+    <>
+      {activeCampaigns[provider] && campaignEndTimeInSeconds >= 1 ? (
+        <Box mt="16px" background="#F0E4E2" padding="16px" border="1px solid #D67E0A" borderRadius="16px">
+          <Flex>
+            <Image src={pocketWatch} alt="pocket-watch" height={30} width={30} />
```

Comment:
I think we can just use `src="/images/pocket-watch.svg"`

Reply:
yeah thats fair saves an import